VTUL/dlp-access
# Sets ebsEnabled to true

**Sets ebsEnabled to true.**
* * *

**JIRA Ticket**: (link) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
OpenSearch requires config to explicitly enable EBS.

# What's the changes? (:star:)
A in-depth description of the changes made by this PR. Technical details and possible side effects.

* Sets ebsEnabled to true in `amplify/backend/api/collectionarchives/override.ts`
* All other changes are a result of creating the `opendev` backend which I'm hoping will soon replace `newdev` and `dlpdev`

# How should this be tested?

* Check that tests are passing an preview is generated
* Visually check over the preview site to make sure that nothing is broken.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

* branch: `ebsEnabled`

# Interested parties
@yinlinchen 

(:star:) Required fields
